### DISCUSSION TITLE
Realm Java with Realm Core 10.5.4

### DISCUSSION THREAD

**Discussion by Riflin** [realm/realm-java#7356] [Q&A] [answered]
Hi! I don't know where to put this, so I'm writing it here... do you have any estimated date to release a version with the core updated to 10.5.4? It's important for my company, as there are some applications still waiting to migrate from realm 7 to realm 10 and we don't want to migrate until the "no such object" problem is solved.

Thanks in advance and sorry if this is not the correct place to answer this!

**Comment by geragray** [comment, marked as answer]
@Riflin Thanks for asking. We should have a release ready by the end of the week at the latest.

**Reply by Riflin** [reply]
Thanks a lot!!! 😄

**Reply by geragray** [reply]
@Riflin we got Java 10.4 ready. Please see https://github.com/realm/realm-java/blob/master/CHANGELOG.md for details.
